src/components/ProjetoCard.tsx

```
@@ -51,37 +53,30 @@ export function ProjetoCard({
-                <p className="text-sm text-gray-700">{projeto.status}</p>
-
-                {/* Horas (abaixo do status) */}
-                <div className="flex items-center gap-2 mt-2">
-                  <Clock size={16} />
-                  <div>
-                    <p className="text-sm font-semibold">Horas</p>
-                    <p className="text-sm text-gray-700">{horas} Horas</p>
-                  </div>
-                </div>
+                <p className="text-sm font-semibold text-nowrap">Vagas</p>
+                <p className="text-sm text-gray-700">{projeto.vagasDisponiveis}</p>
```

ESLint

[error] Replace `{projeto.vagasDisponiveis}` with `⏎··················{projeto.vagasDisponiveis}⏎················`

(prettier/prettier)